src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1186,14 +1197,14 @@ protected static void VerifyOperationTreeForTest<TSyntaxNode>(CSharpCompilation
 
         protected static CSharpCompilation CreateCompilationWithMscorlibAndSpan(string text, CSharpCompilationOptions options = null, CSharpParseOptions parseOptions = null)
```

Comment:
Which TF does this target? Is it portable?

Reply:
Blah I missed that beacuse it's in a hidden region.

Reply:
Going to clean that up as a follow up change. Ideally want to just switch to the System.Memory pcakage.